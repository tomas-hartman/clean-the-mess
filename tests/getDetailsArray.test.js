const { getDetailsArray } = require("../src/popup/popup");

const { tabs } = require("./__sampleData__/tabs");
const { tabsOverview } = require("./__sampleData__/tabsOverview");

it('should return array with details only for certain url', () => {
    const overviewId = 0;
    const output = getDetailsArray(0, tabsOverview, tabs);

    expect(output).toMatchSnapshot();
    expect(output.length).toBe(tabsOverview[overviewId].ids.length);
});

// vymyslet failcase scénáře