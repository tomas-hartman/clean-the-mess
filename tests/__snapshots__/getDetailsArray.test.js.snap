// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return array with details only for certain url 1`] = `Array []`;
